libs/core/src/lib/layout-panel/layout-panel.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component, ElementRef, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+import { AbstractFdNgxClass } from '../utils/abstract-fd-ngx-class';
+
+/**
+ * Layout Panels are used to encapsulate part of the content, form elements, lists, collections, etc., on a page.
+ */
+@Component({
+    selector: 'fd-layout-panel',
+    templateUrl: './layout-panel.component.html',
+    host: {
+        '[class.fd-has-display-block]': 'true'
+    },
+    encapsulation: ViewEncapsulation.None,
+    styleUrls: ['./layout-panel.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LayoutPanelComponent extends AbstractFdNgxClass {
```

Comment:
Let's not use `AbstractFdNgxClass` ( I know i was there before, but please replace it by Gracjan's approach with `CssClassBuilder`)

Reply:
I have just renamed this component :) I haven't refactored it